src/applications/vaos/utils/selectors.js

```diff
@@ -430,3 +430,10 @@ export const selectPastAppointments = createSelector(
     return past?.filter(isValidPastAppointment).sort(sortByDateDescending);
   },
 );
+
+export function selectHasExpressCareRequests(state) {
+  return (
+    vaosExpressCare(state) &&
```

Comment:
Including the feature flag in here simplified the checks in the components for the tabs, even though it's not necessary for the appointments page content above the tab.